spanner/r2dbc/src/main/resources/application.properties

```diff
@@ -0,0 +1,16 @@
+#  Copyright 2021 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+spring.r2dbc.url=\
+r2dbc:spanner://spanner.googleapis.com:443/projects/${project}/instances/${instance}/databases/${database}
```

Comment:
Is the `//spanner.googleapis.com:443` required? For the JDBC driver we have kept that part of the connection string optional, as for most cases the users do not need to change it.